administrate-field-lat_lng.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency 'administrate', '~> 0.2.1'
+  spec.add_dependency 'leaflet-rails', '0.7.7'
```

Comment:
I didn't think too much about it, I just wen't with the version that was used before. Will take a look at `~> 1.0`.